dev/n_osync.sh

```diff
@@ -508,7 +514,7 @@ function _HandleLocksRemote {
 	CheckConnectivityRemoteHost
 
 	# Create an array of all currently running pids
-	read -a initiatorRunningPids <<< $(ps -A | tail -n +2 | awk '{print $1}')
+	read -a initiatorRunningPids <<< $(ps -a | tail -n +2 | awk '{print $1}')
```

Comment:
This is a 'all systems' wide change that can have repercussions on other environments.

Reply:
As per previous comment ( https://github.com/deajan/osync/pull/219#pullrequestreview-523646324 )